5.2.cup_of_join.py

```diff
@@ -1,17 +1,17 @@
-def join(*args, sep='-'):
-    res = []
-    for i, arg in enumerate(args):
-        res.extend(arg)
-        if i < len(args) - 1:
-            res.append(sep)
-
-    if not res:
-        return None
-    else:
-        return res
-
-
-#print(join([1, 2], [8], [9, 5, 6], sep='@'))
-#print(join([1, 2], [8], [9, 5, 6]))
-#print(join([1]))
+def join(*args, sep='-'):
+    res = []
+    for i, arg in enumerate(args):
+        res.extend(arg)
+        if i < len(args) - 1:
+            res.append(sep)
+
+    if not res:
+        return None
+    else:
+        return res
+
+
+#print(join([1, 2], [8], [9, 5, 6], sep='@'))

```

Comment:
Why document your tests?
You should not have any unnecessary documentations in your code.
either it is a working line of code or an explanation, that's it.